app/assets/components/ui-elements/oj-annotation.html

```diff
@@ -1,27 +1,41 @@
 <polymer-element name='oj-annotation' attributes="annotation color paper">
   <template>
     <div class='container' style=" background-color: {{color}}" >
-      <div class='content'>
-        <p>{{annotation.body}}</p>
-      </div>
 
-      <div class='comments'>
-        <template repeat="{{response in annotation.responses}}">
-          <p>{{response.body}}</p>
+      <div  on-tap="{{toggleCollapsed}}" class='content'>
+        <template if="{{collapsed}}">
+          <p layout horizontal>
+            <span> {{annotation.body | truncate}}</span>
+            <span flex></span>
+            <span>{{annotation.responses | count}} replies</span>
```

Comment:
Great will change that up and push
On Tue, Feb 17, 2015 at 8:12 PM Marc Rohloff <notifications@github.com>
wrote:

> In app/assets/components/ui-elements/oj-annotation.html
> <https://github.com/openjournals/theoj/pull/123#discussion_r24874673>:
>
> >
> > -      <div class='comments'>
> > -        <template repeat="{{response in annotation.responses}}">
> > -          <p>{{response.body}}</p>
> > +      <div  on-tap="{{toggleCollapsed}}" class='content'>
> > +        <template if="{{collapsed}}">
> > +          <p layout horizontal>
> > +            <span> {{annotation.body | truncate}}</span>
> > +            <span flex></span>
> > +            <span>{{annotation.responses | count}} replies</span>
>
> I tested and it works for me without the count hack.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openjournals/theoj/pull/123/files#r24874673>.
>